src/main/java/oop/Casting.java

```diff
@@ -0,0 +1,32 @@
+package main.java.oop;
+
+public class Casting {
+    public static void main(String[] args) {
```

Comment:
you don't need main method here. just put methods in this class, and call them from your App class